pkg/cmd/admin/policy/modify_roles.go

```diff
@@ -336,6 +342,19 @@ func (o *RoleModificationOptions) AddRole() error {
 	if err != nil {
```

Comment:
you only need to look up existing rolebindings if they didn't provide a rolebinding name